services/agents/registry.go

```diff
@@ -1004,6 +1004,41 @@ func (r *Registry) StartPTSummaryAction(ctx context.Context, id, pmmAgentID stri
 	return nil
 }
 
+// StartPTMySQLSummaryAction starts pt-mysql-summary action on the pmm-agent.
+// The pt-mysql-summary may require some of the following params:
+// address	IP address
+// port
+// socket	unix socket
+// username
+// password
+// return:		nil - ok, otherwise an error code
```

Comment:
We never format comments like that

Reply:
Another thing that is to be defined in the coding standard. Can you type here, how would you like to have it formatted? I mean you can take this one and reformat. Or if you have a template, it would be enough.

Reply:
https://golang.org/doc/effective_go.html#commentary
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
Both links are already present on the coding standard page.

Try to run godoc and see how it being rendered.

And can you please looks around at the existing code?